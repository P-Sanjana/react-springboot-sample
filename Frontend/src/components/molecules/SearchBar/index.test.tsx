import React from 'react';
import { render } from '@testing-library/react';
import SearchComponent from './index';

describe('Search bar Component', () => {
  it('should match snapshot', () => {
    const wrapper = render(
      <SearchComponent label='Search all assests' variant='outlined' />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
