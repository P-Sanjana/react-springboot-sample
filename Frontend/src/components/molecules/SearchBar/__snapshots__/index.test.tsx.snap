// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search bar Component should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root MuiBox-root-3 makeStyles-textField-1"
      >
        <div
          class="MuiBox-root MuiBox-root-4"
        >
          <div
            class="MuiInputBase-root MuiInputBase-colorPrimary css-1le28he-MuiInputBase-root"
          >
            <input
              class="MuiInputBase-input css-yz9k0d-MuiInputBase-input"
              placeholder="Search all assests"
              type="text"
              value=""
            />
          </div>
        </div>
        <button
          aria-label="search"
          class="MuiButtonBase-root MuiIconButton-root"
          tabindex="0"
          type="submit"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-searchIcon-2"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root MuiBox-root-3 makeStyles-textField-1"
    >
      <div
        class="MuiBox-root MuiBox-root-4"
      >
        <div
          class="MuiInputBase-root MuiInputBase-colorPrimary css-1le28he-MuiInputBase-root"
        >
          <input
            class="MuiInputBase-input css-yz9k0d-MuiInputBase-input"
            placeholder="Search all assests"
            type="text"
            value=""
          />
        </div>
      </div>
      <button
        aria-label="search"
        class="MuiButtonBase-root MuiIconButton-root"
        tabindex="0"
        type="submit"
      >
        <span
          class="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root makeStyles-searchIcon-2"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
